md/custom-widgets.md

```diff
@@ -21,6 +21,12 @@ For more information, see the AngularJS guide on [templates](https://docs.angula
 
 For now, the definition of custom containers is not yet available in the product.
 
+::: warning
+Note that UI Designer always prefix the ID of the custom widgets by "custom", 
+whereas the [standard widgets](widgets.md) are internally using the prefix "pb".
+If you build custom widgets outside of the UI Designer, your widget ID must not start with "pb".
```

Comment:
Maybe (since the second part of the sentence is singular)
"If you build **a widget** outside..."